lib/lib.js

```diff
@@ -74,7 +74,12 @@ function maybeTestForPlusOne(config, comment) {
   return /\+1/.test(comment);
 }
 
+exports.formatReviewedByMetaData = formatReviewedByMetaData;
 function formatReviewedByMetaData(reviewer) {
+  if (!reviewer.email) {
```

Comment:
oh and i think it is a patchlevel version bump - in my eyes the `<null>` was a bug